src/test/java/com/google/api/generator/gapic/model/MessageTest.java

```diff
@@ -0,0 +1,110 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.gapic.model;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertThrows;
+
+import com.google.api.generator.engine.ast.TypeNode;
+import com.google.api.generator.engine.ast.VaporReference;
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import org.junit.Test;
+
+public class MessageTest {
+
+  public static final String MESSAGE_NAME = "TestMessage";
+  public static final Message.Builder TEST_MESSAGE_BUILDER =
```

Comment:
A builder is probably a mutable instance. Generally don't make it look like a constant for non-constants.
```suggestion
  public static final Message.Builder testMessageBuilder =
```
I wouldn't mind dropping `static` as well.

Reply:
In my opinion, as long as the field is final(not reassignable), then it's a constant, but I do see that Java style guide says differently go/java-style#s5.2.4-constant-names. I can change it to follow our style guide.